tests/functional/disaster-recovery/regional-dr/test_failover_and_relocate_discovered_apps.py

```diff
@@ -28,67 +27,85 @@ class TestFailoverAndRelocateWithDiscoveredApps:
     """
 
     @pytest.mark.parametrize(
-        argnames=["primary_cluster_down", "pvc_interface", "kubeobject", "recipe"],
+        argnames=[
+            "primary_cluster_down",
+            "pvc_interface",
+            "kubeobject",
+            "recipe",
+            "iterations",
+        ],
         argvalues=[
             pytest.param(
                 False,
                 constants.CEPHBLOCKPOOL,
-                0,
                 1,
-                id="primary_up-rbd-recipe",
+                1,
+                1,
+                marks=tier1,
+                id="primary_up-rbd-recipe-resourcelabel",
             ),
             pytest.param(
                 True,
                 constants.CEPHBLOCKPOOL,
-                0,
                 1,
-                id="primary_down-rbd-recipe",
+                1,
+                1,
+                marks=tier4,
+                id="primary_down-rbd-recipe-resourcelabel",
             ),
             pytest.param(
                 False,
                 constants.CEPHBLOCKPOOL,
                 1,
-                0,
-                id="primary_up-rbd-kubeobject",
+                1,
+                3,
+                marks=tier4,
+                id="primary_up-rbd-multiple-iterations",
             ),
             pytest.param(
                 True,
                 constants.CEPHBLOCKPOOL,
                 1,
-                0,
-                id="primary_down-rbd-kubeobject",
+                1,
+                3,
+                marks=tier4,
+                id="primary_down-rbd-multiple-iterations",
```

Comment:
kubeobject term is missing from id for all params where it was before the change